skimage/measure/_structural_similarity.py

```diff
@@ -6,12 +6,13 @@
 from ..util.dtype import dtype_range
 from ..util.arraycrop import crop
 from .._shared.utils import deprecated
+from .._shared.utils import skimage_deprecation, warn
 
 __all__ = ['compare_ssim']
 
 
 def compare_ssim(X, Y, win_size=None, gradient=False,
-                 dynamic_range=None, multichannel=False,
+                 data_range=None, dynamic_range=None, multichannel=False,
```

Comment:
Please, move `dynamic_range=None` to the end (before **kwargs), so to keep the same positional signature.